src/Pages/Station/Station.tsx

```diff
@@ -1,31 +1,45 @@
-import * as classNames from 'classnames';
+import { Station as StationModel } from 'Models/Station';
 import { AddSong, StationBrowser } from 'Modules/Station';
 import * as React from 'react';
 import { Component } from 'react';
-import { Col, Row } from 'reactstrap';
+import { RouteComponentProps } from 'react-router-dom';
+import { Col, Container, Row } from 'reactstrap';
 import './Station.scss';
 
 import {
   NowPlaying,
   PlaylistTabs,
   StationHeader,
 } from 'Modules/Station';
+import { StationServices } from 'Services/Http';
+import { StationPlaylistSSE } from 'Services/SSE';
+import './Station.scss';
 
 interface IProps {} // tslint:disable-line
 
 interface IState {
   muted: boolean;
   isPassive: boolean;
+  station: StationModel;
 }
 
-export class Station extends Component<IProps, IState> {
-  constructor(props: IProps) {
+export class Station extends Component<
+  IProps & RouteComponentProps<any>,
+  IState
```

Comment:
Can you make this component stateless? I don't want to call service in Pages, just in Modules

Reply:
I have fixed it!